### PR TITLE
Feature: GraphQL Client

### PR DESCRIPTION
**Pull request by BrennanAyers**
# Description
- [x] Version updated in setup.py
## Briefly describe what this PR accomplishes and what the definition of done is.
- This PR brings in a new GraphQL client allows a user to easily interact with a GraphQL endpoint and work with the resulting response.
- The client takes in an instance of an `HttpClient` to perform queries
- On a successful request, a `GraphQlResponse` object is returned with `http_response`, `data`, and `data_as_structure` properties
- On a failed request, a `QueryFailed` exception is raised with `http_response`, `query`, `query_variables`, `data`, and `errors` properties
- Added `parameterized` to the project requirements for use in unit tests
## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a migration

### CODE REVIEW

File: questions_three/graphql/graphql_client.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    def execute(self, operation, **variables):
+        """
+        :param operation: A string literal containing either a GraphQL query or mutation
+        :param variables: Key=value pairs representing required variables for the GraphQL operation
+        :return: A GraphQlResponse object on a successful request, an OperationFailed object when errors are detected in
+        the response
+        """
+        response = self.http_client.post(
+            url=self.url,
+            headers={'Content-Type': 'application/json'},
```
Comment (ToddBradley) [inline questions_three/graphql/graphql_client.py:22]: When should we run all this through Black?

**Comment by ToddBradley** [comment]
My opinion is to do a squash merge when you merge this to master.

File: questions_three/graphql/graphql_client.py (hunk 1 above)
Comment (BrennanAyers) [reply questions_three/graphql/graphql_client.py:22]: I guess that would depend on the rest of the repo getting run as well. So, I would imagine, after it gets merged in? :)

Comment (ToddBradley) [reply questions_three/graphql/graphql_client.py:22]: Alright, the answer for this is now settled. @egabe05 will Black-ify separately, as part of https://cybergrx.atlassian.net/browse/INT-138.

**Pull request merged**